app/src/debug/java/net.squanchy/DebugActivity.java

```diff
@@ -62,11 +61,8 @@ private void createAndNotifyTalksCount(int count) {
     }
 
     private Event createTestEvent(int id) {
-        Calendar calendar = Calendar.getInstance();
-        calendar.add(Calendar.MINUTE, 5);
-        DateTime start = new DateTime(calendar.getTime());
-        calendar.add(Calendar.MINUTE, 45);
-        DateTime end = new DateTime(calendar.getTime());
+        DateTime start = new DateTime().plusMinutes(5);
+        DateTime end = new DateTime().plusMinutes(45);
```

Comment:
This is _technically_ different from what you had before (the new date will be a few millis after the first one) but hey it's test code XD